tests/test_authentication.py

```diff
@@ -130,7 +130,7 @@ async def app(request):
 @app.route("/dashboard/decorated")
 @async_inject_decorator(additional="payload")
 @requires("authenticated")
-async def decorated_sync(request, additional):
+async def decorated_async(request, additional):
```

Comment:
These two functions were redefinitions which flake8 picked up. I tried to give them better names.